Classes/YLTableView.m

```diff
@@ -71,7 +73,7 @@ - (void)registerClass:(Class)headerFooterViewClass forHeaderFooterViewReuseIdent
   }
 }
 
-- (YLTableViewCell *)sizingCellForReuseIdentifier:(NSString *)reuseIdentifier {
+- (nonnull YLTableViewCell *)sizingCellForReuseIdentifier:(nonnull NSString *)reuseIdentifier {
   NSAssert(reuseIdentifier != nil, @"Must have a reuse identifier.");
```

Comment:
- It seems like we're checking for `x` instead of `x != nil` in other places. Shall we do the same for reuseIdentifier?
- Also do we still need this `NSAssert` now that the `reuseIdentifier` is `nonnull`.